app/views/coronavirus_form/check_answers.html.erb

```diff
@@ -15,21 +15,16 @@
 
 <%= tag.p t("check_your_answers.description"), class: "govuk-body" %>
 
-<%= render "govuk_publishing_components/components/summary_list", {
-  items: items_part_1,
-  block: sanitize(
-          tag.p(
-            link_to("Add an additional product",
-                    "/medical-equipment-type",
-                    class: "govuk-link"
-            ),
-          class: "govuk-body app-body--right")
-         )
-} %>
-
-<%= render "govuk_publishing_components/components/summary_list", {
-  items: items_part_2,
-} %>
+<% form_outline[:properties].each do |section, property| %>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: section.gsub("-", " ").humanize,
+    margin_bottom: 3,
+  } %>
+  <%= render "govuk_publishing_components/components/summary_list", {
```

Comment:
Summary list component was updated to allow [custom headings](https://components.publishing.service.gov.uk/component-guide/summary_list/with_custom_section_heading) in preparation for this change. For consistency, we should try to use the `title` and `heading_level`/`heading_size` properties instead of a separate heading component.

Reply:
Thanks @alex-ju 👍  Have updated to use `summary_list` component where possible.